### DISCUSSION TITLE
Altmetric not showing for a paper

### DISCUSSION THREAD

**Discussion by soumik-kanad** [alshedivat/al-folio#1732] [Q&A] [answered]
Altmetric works for other paper entries but not shown for the following entry even though the altmetric page exists ([https://www.altmetric.com/details/153177353](https://www.altmetric.com/details/153177353)):
```
@article{mukhopadhyay2023diff2lip,
title={Diff2Lip: Audio Conditioned Diffusion Models for Lip-Synchronization}, 
author={Soumik Mukhopadhyay and Saksham Suri and Ravi Teja Gadde and Abhinav Shrivastava},
year={2023},
journal={Under Review},
eprint={2308.09716},
archivePrefix={arXiv},
primaryClass={cs.CV},
doi = {10.48550/arXiv.2308.09716},
arxiv = {2308.09716},
website = {https://soumik-kanad.github.io/diff2lip/},
pdf = {https://arxiv.org/pdf/2308.09716.pdf},
selected = {true},
preview = {diff2lip.png}
}
```
Can anyone help me with this?

One thing that I observed on [https://www.altmetric.com/details/153177353](https://www.altmetric.com/details/153177353) is that the Publisher website links to [https://arxiv.org/abs/2308.09716v1](https://arxiv.org/abs/2308.09716v1) which has a 'v1' at the end. Do you think that might be what is causing the problem? If so, how can that be solved?

**Comment by george-gca** [comment]
Try to add the altmetric id to the bib information, like it is done for the Eistein paper [here](https://github.com/alshedivat/al-folio/blob/cba3ad27fb3609910ad302df58f43e5bf38a42dd/_bibliography/papers.bib#L45). Also see that it has an [order of precedence](https://github.com/alshedivat/al-folio/blob/cba3ad27fb3609910ad302df58f43e5bf38a42dd/_layouts/bib.html#L191) of what information use from bib to create your altimetric badge. Change to use altmetric id first in your case, see if that fixes it.

**Reply by soumik-kanad** [reply]
I tried the Einstein paper (removed the doi) and only used the alt metric 
```
@article{PhysRev.47.777,
  abbr={PhysRev},
  title={Can Quantum-Mechanical Description of Physical Reality Be Considered Complete?},
  author={Einstein, A. and Podolsky, B. and Rosen, N.},
  abstract={}  
  journal={Phys. Rev.},
  volume={47},
  issue={10},
  pages={777--780},
  numpages={0},
  year={1935},
  month={May},
  publisher=aps,
  url={http://link.aps.org/doi/10.1103/PhysRev.47.777},
  html={https://journals.aps.org/pr/abstract/10.1103/PhysRev.47.777},
  pdf={example_pdf.pdf},
  altmetric={248277},
  dimensions={true},
  selected={true}
}
```
This works 
<img width="651" alt="image" src="https://github.com/alshedivat/al-folio/assets/14940967/b2eb4cb1-3132-4356-8de3-ce0a558c16b3">

But when I change the `altmetric={153177353}`, it stops working 
<img width="613" alt="image" src="https://github.com/alshedivat/al-folio/assets/14940967/d5954dc6-159f-4d27-9257-afd4e4a2ecbf">

**Reply by george-gca** [reply, marked as answer]
I was able to make your altmetric work, and it worked both with altmetric id or arxiv (eprint). The thing that we were missing is this:

```html
<span ... data-hide-less-than="15"
```

in [_layouts/bib.html](https://github.com/alshedivat/al-folio/blob/cba3ad27fb3609910ad302df58f43e5bf38a42dd/_layouts/bib.html#L192C45-L192C45) currently it is set to ignore data with less than 15 citations, which is your case. Just lower that number 15 to 1, for example, and it will work.

Also, idk why, but when you change your bib information and you are running locally, you should stop your jekyll process and run it again so it updates it correctly. So yes, it is working for the Einstein paper with only the altmetric id, if you rerun `bundle exec jekyll serve --lsi`.

**Reply by soumik-kanad** [reply]
Thank you @george-gca. This worked!
